spec/controllers/idv/capture_doc_controller_spec.rb

```diff
@@ -0,0 +1,125 @@
+require 'rails_helper'
+
+describe Idv::CaptureDocController do
+  include DocAuthHelper
+  include DocCaptureHelper
+
+  describe 'before_actions' do
+    it 'includes corrects before_actions' do
+      expect(subject).to have_actions(:before,
+                                      :ensure_user_id_in_session,
+                                      :fsm_initialize,
+                                      :ensure_correct_step)
+    end
+  end
+
+  let(:user) { create(:user) }
+  token = nil
+
+  before do
+    enable_doc_auth
+    stub_analytics
+    allow(@analytics).to receive(:track_event)
+    capture_doc = CaptureDoc::CreateRequest.call(user.id)
+    token = capture_doc.request_token
+  end
```

Comment:
```suggestion
  end
  
  let(:capture_doc) { CaptureDoc::CreateRequest.call(user.id) }
  let!(:token) { capture_doc.request_token }
```

I like to have values that are used in tests be defined using `let` and `let!` rather than depending on variable capture.